linera-views/src/map_view.rs

```diff
@@ -1380,7 +1430,7 @@ where
     C: Context + Sync,
     ViewError: From<C::Error>,
     I: CustomSerialize,
-    V: Clone + DeserializeOwned + 'static,
+    V: Clone + DeserializeOwned + Send + 'static,
```

Comment:
Ideally, this would be added for the smallest `impl` section that requires it.

Reply:
I found a way to avoid having this added requirement on the type.